source/tutorials/hpc.rst

```diff
@@ -0,0 +1,688 @@
+.. _hpc:
+
+A Simple HPC Cluster on |CL|
+############################
+
+This tutorial demonstrates how to set a simple :abbr:`HPC (High
+Performance Computing)` cluster using `slurm`_, `munge`_, and 
+`PDSH`_.  For this tutorial, this cluster consists of a controller node 
+and four worker nodes, as shown in Figure 1.  For the sake of simplicity,
+each node resides on a separate host and their hostnames are hpc-controller,
+hpc-worker1, hpc-worker2, hpc-worker3, and hpc-worker4.  
+
+.. rst-class:: dropshadow
+
+.. figure:: /_figures/hpc/hpc-01.png
+   :alt: Simple HPC cluster
+
+   Figure 1: Simple HPC cluster
+
+The configuration is intentionally kept simple, notably avoiding setting 
+up cgroups and accounting. These and many more additional configuration 
+options can be added later.
+
+Prerequisites
+*************
+
+* Knowledge and experience with HPC 
+* Familiarity with slurm, munge, and PDSH
+* All nodes have synchronized clocks (typically by NTP)
+
```

Comment:
```suggestion
.. caution::
   Modifying the `hostname` of a pre-existing host may potentially cause networking errors.   
   Therefore, we recommend starting with a new installation to achieve the best results.
```

Reply:
Since we also set the assumption of fresh installation at the intro, I do not think this "caution" is necessary.

Reply:
That's fine.

Reply:
@jurobystricky also commented on this below.